### PR TITLE
Updated QuickStart Guide

### PR DESCRIPTION
**Pull request by vikash1810**
## Changelog
- Added a FAQ section at the end
- Added alternative of ngrok for loophole
- Added explanation for --stage parameter
- Added steps to integrate telegram bot
- Minor Fixes

### CODE REVIEW

File: docs/building-with-jugalbandi/technical-guide/tutorials/quickstart.md [hunk 1]
```diff
@@ -11,7 +11,8 @@ This quickstart is designed for linux based systems such as Linux distros and Ma
 1. **Docker** - Ensure that your system has docker engine installed and running. For installation, please refer to [docker engine installation instruction](https://docs.docker.com/engine/install/).
    * The recommended way is to install docker desktop for your system. If using WSL with Windows, Enable docker for you WSL distro. Please refer to [docker desktop installation](https://learn.microsoft.com/en-us/windows/wsl/tutorials/wsl-containers#install-docker-desktop) for more details.
 2. **Docker Compose** - Ensure docker compose is enabled along with docker engine. Please refer to [docker compose installation instruction](https://docs.docker.com/compose/install/).
-3. **Loophole** - Loophole is required to expose your local system to the services on web by tunneling the external calls to your local system. Please refer to [Loophole Quickstart Guide](https://loophole.cloud/download/) for installation.
+3. **Loophole** - Loophole is required to expose your local system to the services on the web by tunneling the external calls to your local system. Please refer to the [Loophole Quickstart Guide](https://loophole.cloud/download/) for installation.
```
Comment (KaranrajM) [inline docs/building-with-jugalbandi/technical-guide/tutorials/quickstart.md:14]: Can we make both loophole and ngrok as subpoints just to tell them apart as either/or and give a heading to point 3 (eg: tunnel)?

File: docs/building-with-jugalbandi/technical-guide/tutorials/quickstart.md [hunk 2]
```diff
@@ -72,9 +73,15 @@ $ python3 -m http.server 5000
 
 6. Start a new shell session and start loophole tunnel for port 5000 on your system
```
Comment (KaranrajM) [inline docs/building-with-jugalbandi/technical-guide/tutorials/quickstart.md:74]: Can we pls remove loophole word here as it is just tunnel?

File: docs/building-with-jugalbandi/technical-guide/tutorials/quickstart.md [hunk 3]
```diff
@@ -72,9 +73,15 @@ $ python3 -m http.server 5000
...
+     ```bash
+     $ ./loophole http 5000
+     ```
+
+   - For **ngrok**, use the following command:
+     ```bash
+     $ ngrok http 5000
+     ```
 
 Copy the tunnel url from the loophole shell and add it to `PUBLIC_URL_PREFIX` in `.env-dev` file. ![](../../../assets/quickstart-loophole-5000.png)
```
Comment (KaranrajM) [inline docs/building-with-jugalbandi/technical-guide/tutorials/quickstart.md:86]: We can remove loophole and just keep the shell or we can say something like (copy the tunnel url from the example loophole shell shown below)

File: docs/building-with-jugalbandi/technical-guide/tutorials/quickstart.md [hunk 4]
```diff
@@ -110,14 +128,20 @@ $ bash scripts/run.sh --stage api channel language flow frontend
 5. Once the above steps are completed, the bot status will be changed from **inactive** to **active**. ![](../../../assets/quickstart-activebot.png)
 6. Start a new shell session and start loophole tunnel for port 8000 on your system
```
Comment (KaranrajM) [inline docs/building-with-jugalbandi/technical-guide/tutorials/quickstart.md:129]: Same comment as above, we can remove loophole word here.

File: docs/building-with-jugalbandi/technical-guide/tutorials/quickstart.md [hunk 5]
```diff
@@ -136,3 +160,52 @@ $ ./loophole http 8000
     --data-raw "$BODY"
     ```
 9. Your bot is running. Send a `Hi` message to whatsapp business number to start conversation with the bot.
+
+### Quickstart FAQs
```
Comment (KaranrajM) [inline docs/building-with-jugalbandi/technical-guide/tutorials/quickstart.md:164]: Can we order the FAQs by giving numbers? Will it be better?

**Review by KaranrajM — COMMENTED** [review]
Hi @vikash1810 I have left some comments after reviewing. Please have a look at it and let me know what you think.

**Comment by vikash1810** [comment]
Hey @KaranrajM, I'll integrate the suggestions in the next commit.

**Comment by KaranrajM** [comment]
Hi @vikash1810 the changes look great. One final suggestion is to replace the telegram integration steps in FAQs with "The telegram integration documentation and frontend is still being worked on" as we want the telegram integration documentation to be a separate page. Also please check the layout of FAQs in the markdown format, the numbering does not work after 1. Please make these changes

**Comment by vikash1810** [comment]
Hey @KaranrajM, I've made the suggested changes in the latest commit. Please go through it.

**Pull request merged**